### PR TITLE
Update PHPUnit to 9.1

### PR DESCRIPTION
**Pull request by shuchkin**
Screenshot_294

Update PHPUnit to 9.1
Update react/dns to 1.x

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -15,14 +15,16 @@
...
     "require": {
         "php": "^7.0",
-        "react/event-loop": "^1.0.0",
-        "react/socket": "^1.0.0",
+        "react/event-loop": "^1.0",
+        "react/socket": "^1.0",
         "nekland/tools": "^1.0 || ^2.0",
         "psr/log": "^1.0",
-        "react/dns": "^0.4.6"
+        "react/dns": "^1.0"
```
Comment (Nek-) [inline composer.json:22]: This looks great, other versions were good.

File: composer.json [hunk 2]
```diff
@@ -15,14 +15,16 @@
...
     },
     "require-dev": {
-        "phpunit/phpunit": "^8.0"
+        "phpspec/prophecy-phpunit": "^2.0",
+        "phpunit/phpunit": "^9.1",
+        "ext-soap": "*"
```
Comment (Nek-) [inline composer.json:27]: soap is definitely not required :) .

**Review by Nek- — APPROVED** [review]
This is great. How did I miss that? Thanks.

**Review by Nek- — CHANGES_REQUESTED** [review]
Sorry I though I would be able to push to your PR but I'm not, please change what I noticed. 🙏

**Comment by shuchkin** [comment]
check it

**Review by Nek- — APPROVED** [review]

**Comment by Nek-** [comment]
@shuchkin for your information, `^1.0` is the same as `^1.0.0` :) . But nvm lets go for it.

**Pull request merged**